src/renderer/account/components/AccountPanel/index.js

```diff
@@ -10,6 +10,25 @@ import withNetworkData from 'shared/hocs/withNetworkData';
 
 import AccountPanel from './AccountPanel';
 
+const fetch = require('isomorphic-fetch');
+
+const tokenListUrl = 'https://raw.githubusercontent.com/CityOfZion/neo-tokens/master/tokenList.json';
+
+let tokens = [];
+fetch(tokenListUrl).then((data) => data.json()).then((data) => {
```

Comment:
I'd prefer if we stick with async/await syntax within the app for consistency.  We can move this into a function to do so, e.g.:

```js
const tokens = (async () => {
  const result = await fetch(tokenListUrl);
  const data = result.json();
  // etc;
})();
```

Reply:
then tokens is not an array anymore but a Promise. how I should pass it's value to `withInitialCall` then? I'm not able to do `await` there.